MAVProxy/modules/mavproxy_help.py

```diff
@@ -45,13 +45,15 @@ def __init__(self, mpstate):
             self.wxVersion = ''
 
         #check for updates, if able
-        import pip
         pypi = xmlrpclib.ServerProxy('https://pypi.python.org/pypi')
-        available = pypi.package_releases('MAVProxy')
-        if not available:
+        try:
+            available = pypi.package_releases('MAVProxy')
+            if not available:
+                self.newversion = 'Error finding update'
```

Comment:
Setting this to `None` outside here would be better.

So `if available:` sort of thing.

Checking for `None` would then be done down in the "format the update string" step.

Reply:
Fixed